pyproject.toml

```diff
@@ -12,7 +12,7 @@ dependencies = [
     "google-cloud-alloydb-connector[asyncpg]>=1.0.0, <2.0.0",
     "langchain-core>=0.1.1, <1.0.0",
     "langchain-community>=0.0.18, <1.0.0",
-    "numpy==1.24.4; python_version<='3.8'",
+    "numpy==1.26.4; python_version<='3.8'",
```

Comment:
@averikitsch i'm assuming we probably need this to stay `1.24.4`? I wonder how we can have renovate ignore this line...

Reply:
That is correct. I haven't found a solution for pinning it in the pyproject.toml. Overall, I think I need to update the config to not update the minimum version in pyproject.toml

Reply:
Manually updating. Will debug in b/343988502